source/components/molecules/DateTimePicker/DateTimePickerForm.js

```diff
@@ -21,49 +22,63 @@ const DateTimePickerForm = props => {
 
   if (inputValue) {
     switch (mode) {
-      case 'date':
-        dateTimeString = dateString;
+      case 'datetime':
+        dateTimeString = `${dateString} ${timeString}`;
         break;
 
       case 'time':
         dateTimeString = timeString;
         break;
 
       default:
-        dateTimeString = `${dateString} ${timeString}`;
+        dateTimeString = dateString;
     }
   }
 
   return (
-    <ChatForm
-      {...includePropetiesWithKey(props, ['isFocused', 'changeHandler', 'inputValue'])}
-      submitHandler={() => submitHandler(dateTimeString)}
-      renderFooter={() => (
+    <View>
+      <TouchableOpacity onPress={() => setIsVisible(!isVisible)}>
+        <Input
+          placeholder="åååå-mm-dd"
+          editable={false}
+          value={dateTimeString}
+          pointerEvents="none"
+          {...props}
+        />
+      </TouchableOpacity>
+      {isVisible && (
         <DateTimePicker
           value={date}
-          onChange={(_event, value) => changeHandler(value)}
+          onChange={(_event, value) => onSelect(value)}
           mode={mode}
           {...selectorProps}
         />
       )}
-    >
-      <Input
-        placeholder="Välj ett datum"
-        {...props}
-        editable={false}
-        value={dateTimeString}
-        onSubmitEditing={() => submitHandler(dateTimeString)}
-      />
-    </ChatForm>
+    </View>
   );
 };
 
 DateTimePickerForm.propTypes = {
-  changeHandler: PropTypes.func,
+  /**
+   * Function to set input date value
+   */
+  onSelect: PropTypes.func,
+  /**
+   * Value from the date picker
+   */
   inputValue: PropTypes.oneOfType([PropTypes.instanceOf(Date), PropTypes.string]),
+  /**
+   * Defines the type of date picker
+   */
   mode: PropTypes.string,
+  /**
+   * Is an object that can define the locale, minuteInterval or minimumDate
+   */
   selectorProps: PropTypes.object,
-  submitHandler: PropTypes.func,
+  /**
+   * Sets the color theme, default is light.
+   */
+  color: PropTypes.string,
```

Comment:
Very minor, but if you say that the default it light, then make it so? Now it's dependent on the Input component. Also, we could restrict the allowed values to the color schemes defined for the input component.